hazelcast/src/test/java/com/hazelcast/test/HazelcastSerialClassRunner.java

```diff
@@ -48,8 +51,14 @@ protected void runChild(FrameworkMethod method, RunNotifier notifier) {
             long start = System.currentTimeMillis();
             System.out.println("Started Running Test: " + testName);
             super.runChild(method, notifier);
-            float took = (float) (System.currentTimeMillis() - start) / 1000;
-            System.out.println(String.format("Finished Running Test: %s in %.3f seconds.", testName, took));
+            float tookSeconds = (float) (System.currentTimeMillis() - start) / 1000;
+            System.out.println(String.format("Finished Running Test: %s in %.3f seconds.", testName, tookSeconds));
+
+            Category classAnnotations = method.getDeclaringClass().getAnnotation(Category.class);
+
+            if (classAnnotations != null && ArrayUtils.contains(classAnnotations.value(), QuickTest.class)) {
```

Comment:
note that code-coverage tests are significantly slower - we might get many warnings for them